tests/Scripting/TasksDate.test.ts

```diff
@@ -105,3 +105,54 @@ describe('TasksDate', () => {
         expect(new TasksDate(moment('2023-02-31')).fromNow.groupText).toEqual('%%320230611%% Invalid date');
     });
 });
+
+describe('TasksDate - postpone', () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date('2023-11-28'));
+    });
+
+    afterEach(() => {
+        jest.useRealTimers();
+    });
+
+    it('should postpone an older date (before yesterday) to today', () => {
+        const tasksDate = new TasksDate(moment('2023-11-20'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
```

Comment:
Is this postpone() function actually used?

I wonder if it is a remnant of the days when the only option was to postpone one day.

Reply:
I'm wrong, it is used.

I was lulled in to thinking it didn't take any arguments because it provided default values.

### For future consideration...

Personally, I would make the tests more specific, and remove the default values.

If it is useful to have a convenience to postpone by one day, I would then add a convenience wrapper `postponeByOneDay()`.

Reply:
It is not used elsewhere. As far as I understood, the `postpone()` added recently.

Also on the topic - I would remove the default parameter values from `postpone()` at all to avoid confusion.

Reply:
I've removed the default values. Let's discuss the wrapper and the other details during our session.